src/styles/main.scss

```diff
@@ -377,6 +377,12 @@ header.navbar .navbar-brand {
   }
 }
 
+// Custom styles for ng-select validation
+ng-select.ng-invalid.ng-touched .ng-select-container {
+  border-color: #dc3545;
+  box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075), 0 0 0 3px #fde6e8;
```

Comment:
Please, check imported file `_variables.scss` to find out to reusing defined shadow styles.

Reply:
Actually no other input fields use `box-shadow` in invalid styles, so it was removed to keep styles consistent